Update carrier shenanigans information and remove inaccurate information (#171)
PrivSec-dev/privsec.dev: content/posts/android/Android Tips.md

## Carrier Tracking

Carriers can track your coarse location through various means. In order to avoid this type of tracking, you need to do the following:

- Disable Wifi-Calling. Wifi-calling connections do not go through the VPN tunnel, and thus will reveal your Wifi IP address to the carrier.

- Disable the SIMs/eSIMs in **Settings** → **Network & internet** → **SIMs**. On GrapheneOS, if you are using an eSIM, you will need to enable the privileged eSIM management app before doing so. With certain carriers, there will be an ePDG server defined which the operating system will connect to outside of a VPN tunnel. While unlikely, a malicious carrier can track a user by giving them a unique ePDG server. 

- Turn on Airplane mode. This will turn off the baseband modem and your phone will no longer broadcast the IMEI/IMSI to the nearby towers. Note that your phone can still talk to the tower regardless of whether a SIM card inside - this is why `911` calls work. Simply removing the SIM cards is not enough.

> - Disable the eSIM management app after you have disabled all of the eSIMs. With certain carriers, the eSIM management app will connect to the provisioning server to check for eSIM updates. 

Reviewer: Do you mean the _privileged eSIM management toggle_ in GrapheneOS?

- The GOS toggle is very different from disabling an app. Figure out which, if either, achieves the intended result.
- If disabling this app turns out to be impossible on stock OS, then there's no point saying anything other than what works in GOS.
- You need to say what exact Google app is the relevant app here.

Author: It just means the esim management app. I am not sure how it works on random android devices. I guess we can just frame this for GOS for now, what u think?

Reviewer: I still consider the package name/ID required information before merging this.

Author: I don't, this is just gonna sit for eternity before I can pull the correct package ID, and its is different depending on the OS. Not everyone uses the Google one.

> **Note**: I have seen recommendations to use PGPP as a carrier to randomize the IMSI by regularly reprovisioning the eSIM. This is unlikely to be beneficial, as the IMEI broadcasted by the baseband modem would remain unchanged, allowing the carriers to track you anyways.

Reviewer: ```suggestion
On a related note, I have seen recommendations to use PGPP as a carrier to randomize the IMSI by regularly reprovisioning the eSIM. This is unlikely to be beneficial, as the IMEI baked into the modem would remain unchanged, allowing carriers to track you anyways.
```

This is not without its caveats:
- Google Fi requires Play Services and the [Fi app](https://play.google.com/store/apps/details?id=com.google.android.apps.tycho&hl=en_US) to work properly. Without Play Services, all of the features mentioned above, along with visual voicemail will not work. SMS messages will have random strings added at the end of each of them.

> - On GrapheneOS, Fi VPN will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future. 

Reviewer: ```suggestion
- On GrapheneOS, Fi&nbsp;VPN will not function. Fi&nbsp;VPN requires privileged integration with the operating system which is unlikely to be supported for the foreseeable future. 
```

> - Google Fi Wifi-calling does not work behind a VPN.

Reviewer: Needs to be more specific. Does not work when _any_ VPN profile is enabled? Does not work when the VPN profile advertises a route for all traffic? Does not work when the OS is configured to block traffic outside of the VPN?

Author: Okay, so
- The Fi app gotta be in the owner profile iirc, dont even mention it
- It only breaks with the killswitch on
- if you dont disable the killswitch, you don't even need split tunneling
